stories/Organism/Blocks/ImageRevelCards/ImageRevelCards.jsx

```diff
@@ -1,23 +1,35 @@
 import React, { useEffect } from 'react';
-import { ImageRevelCard } from '../../../Components/UIcomponents/Cards/ImageRevelCTACard/ImageCard';
 import { Heading } from '../../../Atom/Typography/Heading/Heading';
-import '../../../Templates/PublicationPage/publicationpage.scss';
+import { Ctalink } from '../../../Components/UIcomponents/Buttons/Cta_link/Cta_link';
+import './imagerevelcards.scss';
```

Comment:
@neelesh-tippe Correct the spelling everywhere. It needs to be `reveal` and **not** `revel`. Also can we just keep the name to `imagecard`?